content/hardware/05.nicla/boards/nicla-vision/datasheet/datasheet.md

```diff
@@ -34,13 +34,13 @@ wireless sensor networks, data fusion, artificial intelligence, machine vision
   - HMAC, CMAC, SHA-1, SHA-224/256/384/512 operations
   - HKDF, MIFARE® KDF, PRF (TLS-PSK)
   - Support of main TPM functionalities
-  - Secured flash user memory up to 50kB
+  - Secured flash user memory up to 50 KByte
```

Comment:
Based on the this page on Wikipedia https://en.wikipedia.org/wiki/Kilobyte, it seems that "The internationally recommended unit symbol for the kilobyte is kB." See footnote: https://en.wikipedia.org/wiki/Kilobyte#cite_note-IEC80000-1
So I would stick with kB (instead of KByte). What do you think @martab1994

Reply:
@jhansson-ard Shall we add those units to the style guide for contribution?